runner/src/main.rs

```diff
@@ -62,6 +62,8 @@ struct BuildArgs {
     #[arg(long)]
     /// Path to the configuration file to use
     config: Option<PathBuf>,
+    #[arg(short, long, default_value = "none")]
+    firmware: String,
```

Comment:
Here you can use `Option<String>` instead of relying on a default value of `"none"`.